### DISCUSSION TITLE
A few questions

### DISCUSSION THREAD

**Discussion by AdamJBurns** [msys2/MSYS2-packages#2359] [General]
Hey guys,

Sorry to be a bother, I just have a few questions some I'm not sure are possible.

Firstly when booting Msys2, I get an error at the top every time saying: `bash: cut: command not found`

Secondly, is it possible to change my starting directory as I currently open in `usr/bin` and would like to switch to Desktop?

Thanks heaps,
Adam

**Comment by Biswa96** [comment]
> Firstly when booting Msys2

MSYS2 is not an operating system.

> bash: cut: command not found

Maybe the `$PATH` is not set correctly. I did not see this type of issue from a clean installation.

> change my starting directory as I currently open in usr/bin

Add `cd /usr/bin` in `~/.bashrc` file.

**Comment by cbrt64** [comment]
> error at the top every time saying: `bash: cut: command not found`

Possibly addressed by #2837
